manifests/server/config.pp

```diff
@@ -163,6 +163,17 @@
                   Exec['puppet_server_config-create_ssl_dir'],
                   ],
     }
+  } elsif $::puppet::server::ca_crl_sync {
+    # If not a ca AND sync the crl from the ca master
+    if defined('$::servername') {
+      file { $::puppet::server::ssl_ca_crl:
+        ensure  => file,
+        owner   => $::puppet::server::user,
+        group   => $::puppet::server::group,
+        mode    => '0644',
+        content => file($::settings::cacrl, $::settings::hostcrl, '/dev/null'),
```

Comment:
I can't find anything on ```$::settings```, but that may be my lacking search skills. Which version introduced these?